packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_GDSWInterfacePartitionOfUnity_decl.hpp

```diff
@@ -56,31 +56,32 @@ namespace FROSch {
         
     public:
         
-        typedef typename GDSWInterfacePartitionOfUnity<SC,LO,GO,NO>::CommPtr CommPtr;
+        typedef typename InterfacePartitionOfUnity<SC,LO,GO,NO>::CommPtr CommPtr;
```

Comment:
1. Prefer C++11 `using` syntax to create type aliases.
2. Do these type aliases really need to be public?

Reply:
Thanks. I changed this in the whole code of FROSch.